### ISSUE TITLE
Lack of accessibility tools

### ISSUE THREAD

**Issue by guicavalheiro** [guicavalheiro/notar-e-anotar-mobile#3]
The project don't have any option of accessibility. A good idea of implemantation would be the change of colors of text and images.
